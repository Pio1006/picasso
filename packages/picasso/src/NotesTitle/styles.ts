import { Theme, createStyles } from '@material-ui/core/styles'
import { PicassoProvider } from '@toptal/picasso-provider'

PicassoProvider.override(({ palette }: Theme) => ({}))

export default ({ palette }: Theme) => createStyles({})
