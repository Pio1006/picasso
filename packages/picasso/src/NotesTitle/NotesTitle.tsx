import React, { forwardRef } from 'react'
import { makeStyles, Theme } from '@material-ui/core/styles'
import { BaseProps } from '@toptal/picasso-shared'

import styles from './styles'

export interface Props extends BaseProps {}

const useStyles = makeStyles<Theme>(styles)

export const NotesTitle = forwardRef<HTMLElement, Props>(function NotesTitle(
  props,
  ref
) {
  const classes = useStyles()

  return <div>{props.children}</div>
})

NotesTitle.defaultProps = {}

NotesTitle.displayName = 'NotesTitle'

export default NotesTitle
