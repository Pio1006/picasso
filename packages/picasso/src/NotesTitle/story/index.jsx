import PicassoBook from '~/.storybook/components/PicassoBook'

import { NotesTitle } from '../NotesTitle'

const page = PicassoBook
  .section('Components')
  .createPage(
    'NotesTitle',
    `<-- description -->`
  )

page
  .createTabChapter('Props')
  .addComponentDocs({ component: NotesTitle, name: 'NotesTitle' })

page
  .createChapter()
