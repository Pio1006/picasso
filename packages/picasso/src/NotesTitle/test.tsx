import React, { ReactNode } from 'react'
import { render } from '@toptal/picasso/test-utils'

import NotesTitle from './NotesTitle'

const renderNotesTitle = (children: ReactNode) => {
  return render(<NotesTitle>{children}</NotesTitle>)
}

describe('NotesTitle', () => {
  it('default render', () => {
    const { container } = renderNotesTitle('title')

    expect(container).toMatchSnapshot()
  })
})
