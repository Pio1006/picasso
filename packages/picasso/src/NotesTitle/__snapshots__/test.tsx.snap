// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotesTitle default render 1`] = `
<div>
  <div
    class="Picasso-root"
  >
    <div
      class="makeStyles-root"
    >
      <h4
        class="MuiTypography-root PicassoTypography-headingSmall MuiTypography-h4"
      >
        title
      </h4>
    </div>
  </div>
</div>
`;
